### ISSUE TITLE
Actor Critic neural combine

### ISSUE THREAD

**Issue by RozenAstrayChen** [MorvanZhou/Reinforcement-learning-with-tensorflow#100]
Hello, I have question . If I want combine actor and critic neural, how could  calculate them loss?
nerual like this:

```
with tf.variable_scope('AC'):
            w_initializer = tf.random_normal_initializer(0.0, 0.01)
            l1 = tf.layers.dense(
                inputs=self.s_in,
                units=32,  # number of hidden units
                activation=tf.nn.relu,
                kernel_initializer=tf.random_normal_initializer(0.,
                                                                .1),  # weights
                bias_initializer=tf.constant_initializer(0.1),  # biases
                name='l1')

            l2 = tf.layers.dense(
                inputs=l1,
                units=32,  # number of hidden units
                activation=tf.nn.relu,
                kernel_initializer=tf.random_normal_initializer(0.,
                                                                .1),  # weights
                bias_initializer=tf.constant_initializer(0.1),  # biases
                name='l2')
            # actor
            self.acts_prob = tf.layers.dense(
                inputs=l2,
                units=n_actions,    # output units
                activation=tf.nn.softmax,   # get action probabilities
                kernel_initializer=tf.random_normal_initializer(0., .1),  # weights
                bias_initializer=tf.constant_initializer(0.1),  # biases
                name='acts_prob'
            )

            # critic
            self.v = tf.layers.dense(
                inputs=l2,
                units=1,  # output units
                activation=None,
                kernel_initializer=tf.random_normal_initializer(0., .1),  # weights
                bias_initializer=tf.constant_initializer(0.1),  # biases
                name='V'
            )
`
